src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PublicVisitController.kt

```diff
@@ -18,6 +18,7 @@ import uk.gov.justice.digital.hmpps.visitscheduler.service.VisitService
 
 const val GET_BOOKED_FUTURE_PUBLIC_VISITS_BY_BOOKER_REFERENCE: String = "/public/booker/{bookerReference}/booked/visits/future"
 const val GET_CANCELLED_PUBLIC_VISITS_BY_BOOKER_REFERENCE: String = "/public/booker/{bookerReference}/cancelled/visits"
+const val GET_BOOKED_PAST_PUBLIC_VISITS_BY_BOOKER_REFERENCE: String = "/public/booker/{bookerReference}/past/visits"
```

Comment:
should the endpoint be similar to - /public/booker/{bookerReference}/booked/visits/future i.e. /public/booker/{bookerReference}/booked/visits/past?